README.md

```diff
@@ -59,24 +59,27 @@ Looking for example to use `react-flexbox-grid`? Head over to [react-flexbox-gri
 
 Advanced composition
 -------
-Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you have a `MyFormInput` component that should also act as both a `Row` and a `Col`.
+Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you're using a third party component that accepts a className.
 
 ```js
-import {getRowClassNames, getColClassNames} from 'react-flexbox-grid'
+// a component that accepts a className
+import RandomComponent from 'some-package';
```

Comment:
I would rather keep all the imports here, in total:

```js
import React from 'react';
import { getRowProps, getColumnProps } from 'react-flexbox-grid';
// a component that accepts a className
import SomeComponent from 'some-package';
```

Also, I'd rename `RandomComponent` to `SomeComponent` (like above), to avoid confusion that the component has some kind of randomized behavior.